docs/en_US/getting-started/getting-started.md

```diff
@@ -1,113 +1,247 @@
-# Overview
+# Product Overview
 
-This project provides an operator for managing EMQX clusters on Kubernetes.
+This project provides an Operator for managing EMQX clusters on Kubernetes.
 
-**Note**: EMQX Operator Controller requires Kubernetes v1.20.11 and up.
+## Deploy EMQX Operator
 
-## Running the Operator
+### Prepare the environment
 
-### Prepare
+Before deploying EMQX Operator, please confirm that the following components have been installed:
 
-We use a [cert-manager](https://github.com/jetstack/cert-manager) for provisioning the certificates for the webhook server. You can follow [the cert-manager documentation](https://cert-manager.io/docs/installation/) to install it.
+| Software                | Version Requirements |
+|:-----------------------:|:--------------------:|
+| Kubernetes              | >= 1.20.11           |
+| Helm                    | >= 3                 |
+| cert-manager            | >= 1.1.6             |
+
+We use [cert-manager](https://github.com/cert-manager/cert-manager) to provide certificates to webhook services. You can install it through [cert manager documentation](https://cert-manager.io/docs/installation/).
+
+### Install Helm
+
+1. Install Helm
+
+Helm is a Kubernetes package management tool. Execute the following command to install Helm directly. For more Helm installation methods, please refer to: [Installing Helm](https://helm.sh/zh/docs/intro/install/)
+
+```bash
+curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+```
+
+2. Add Helm chart repository
+
+```bash
+helm repo add emqx https://repos.emqx.io/charts
+helm repo update
+```
 
 ### Install EMQX Operator
 
-1. Install by helm
+```bash
+helm install emqx-operator emqx/emqx-operator --set installCRDs=true --namespace emqx-operator-system --create-namespace
```

Comment:
```suggestion
helm install emqx-operator emqx/emqx-operator --namespace emqx-operator-system --create-namespace
```

Reply:
The default value of `installCRDs` is `true`

Reply:
> The default value of `installCRDs` is `true`

ok